### ISSUE TITLE
Getting #show error when creating new form page, trying redirect_to (form page)

### ISSUE THREAD

**Issue by jimijack-zz** [jimijack/canvache#1]
What user story am I working on?
- Galleries can set up a profile page with a form that collects: name, location, genres supported, years open, hours etc.  outlined in galleries model.

What error am I getting?
![Error Message](https://i.imgur.com/2QqgQG6.png)

What have I already attempted in trying to fix the error?
- Spoke with Phil and Ezra about redirects in order to allow for a user to be directed to this new form.  I created a new profile view to direct them to when signing up assuming they are an admin (gallery rep).
  Instead of being directed to the form its giving me an error on the show method substituting "profile"for the user "id" saying it can't find.

(See below) Under Register, if the boolean is true, render this new form for gallery_rep to build profile. This form will work for the data model "galleries."  Just not understanding the logic for connecting the register form, boolean, and new form if true.

![My Outline](https://i.imgur.com/caoTmmB.jpg)

**Comment by jimijack-zz** [comment]
Side-Bar - Correction...

It redirects to the /users/index page and not the /users/profile page.  I want the index page to be the standard home screen for any login.  New users with the boolean = true need to set up a profile first.

![User Controller](https://i.imgur.com/ecmHV8l.png)

**Comment by ghost** [comment]
@jimijack:
1. add us (or at least me) as collaborators.
2. assign one of us to the issue so that we can see you posted it!

That's why no one responded!

If this guy is old news, please close.

**Comment by jimijack-zz** [comment]
Yes you and I actually wound up figuring it out. Sorry didn't close I'll be sure and handle that first thing in the am. Thx Phil.

James Ewing
